Mathlib/CategoryTheory/Monoidal/Braided/Basic.lean

```diff
@@ -506,6 +507,9 @@ instance : (𝟭 C).Braided where
 
 instance (F : C ⥤ D) (G : D ⥤ E) [F.Braided] [G.Braided] : (F ⋙ G).Braided where
 
+lemma toMonoidal_injective (F : C ⥤ D) : Function.Injective
+    (@Braided.toMonoidal _ _ _ _ _ _ _ _ _ : F.Braided → F.Monoidal) := by rintro ⟨⟩ ⟨⟩ rfl; rfl
+
```

Comment:
I'm not sure if changes to this file fit in this PR.

Reply:
Whoops, accident